core/consensus/component.go

```diff
@@ -382,8 +383,8 @@ func (c *Component) Participate(ctx context.Context, duty core.Duty) error {
 		return nil // No eager consensus for potential no-op aggregation duties.
```

Comment:
I believe this method's godoc needs to change, because we don't check if the timer is Eager anymore.